x/sequencer/types/genesis.go

```diff
@@ -11,5 +13,21 @@ func DefaultGenesis() *GenesisState {
 // Validate performs basic genesis state validation returning an error upon any
 // failure.
 func (gs GenesisState) Validate() error {
+	// Check for duplicated index in sequencer
+	sequencerIndexMap := make(map[string]struct{})
+
+	for _, elem := range gs.SequencerList {
+
+		// FIXME: should run validation on the sequencer objects
+
+		index := string(SequencerKey(elem.SequencerAddress))
+		if _, ok := sequencerIndexMap[index]; ok {
+			return fmt.Errorf("duplicated index for sequencer")
```

Comment:
predefine an error: `ErrDuplicateSequencerIndex = errorsmod.Register(ModuleName, 1019, "duplicate index for sequencer")` in errors.go